src/common/hook/usePublishQueue/index.tsx

```diff
@@ -51,6 +56,8 @@ export const usePublishQueue = (
           .catch((e) => {
             failedJobs.push(index);
             setFailedJobIndex(failedJobs);
+            jobError.push(e);
+            setJobErrors(jobError);
```

Comment:
Will be better to tie both of these together so that one doesn't have to align on the index and error after. I'll probably change the state to something like:

```json
index: 1,
error: <error here>
```

and then to not break things, can have export the same interface for `failedJobIndex` as

```
const failedJobIndex = failedJobs.map(job=>job.index);
```

^ I'm dont think we need to preserve the old `failedJobIndex` interface though, probably can just skip this

Reply:
ok